autoria.py

```diff
@@ -0,0 +1,253 @@
+        for the given example it would be 1.
+
+        Args:
+            item_to_select - string, could be not 100% accurate as it is in the
+                auto.ria.ua lists, e.g. value ''Харьков'' is acceptable,
+                because the parameter is wild-carded in comprasion like
+                ''*Харьков*'', the function will find suitable name
+                ''Харьковская'' and will return its id.
+            items_list - JSON-formatted list of pairs ''name: value'',
+                    obtained from one of the ''get_'' functions.
+
+        Returns:
+            needed item (category, bodystyle, mark etc.) identifyer.
+        """
+        if item_to_select is not None and items_list is not None:
+            for item in items_list:
+                if fnmatch(item['name'], '*' + item_to_select + '*'):
+                    return item['value']
+        else:
+            return None
+
+    def get_average(self, category: str, mark: str, model: str,
+                    bodystyle: str = None, years: list = None,
+                    state: str = None) -> Union[dict, list]:
+        """Compose parameters for GET request and send it.
+
+        Get average price using search parameters.
+
+        Args:
+            category - vehicle type, e.g. ''Легковые''
+            mark - mark, like ''Renault''
+            model - model, like ''Scenic''
+            bodystyle - (optional) bodystyle, e.g. ''Седан''
+            year - (optional) list with start and end manufacturing year,
+                   e.g. ''[2005, 2006]'', also one of years could be
+                   ''None'', e.g. ''[2005, None]''
+                <== note: make this parameter behave like on auto.ria:
+                if there's only start year end=current_year,
+                vice versa - start=1950==>
+            state - (optional) state (city), e.g. ''Харьков''
+
+            The following search parameters are not in use right now.
+            Are to be added.
+            gear_type - (optional) list with gearshift types,
+                        e.g. ''[Механическая, Автоматическая]''
+            fuel - (optional) list with gasoline types,
+                    e.g. ''[Бензин, Газ/Бензин]''
+            drive - (optional) list with drive types,
+                    e.g. ''[Передний, Полный]''
+            'engine': 'engineVolume',
+            'options': 'options',
+            'mileage': 'raceInt',
+            'doors': 'door',
+            'state': 'state_id',
+            'city': 'city_id',
+            'carrying': 'carrying',
+            'seats': 'seats',
+            'color': 'color_id',
+            'custom': 'custom',
+            'damage': 'damage',
+            'credit': 'under_credit',
+            'confiscated': 'confiscated_car',
+            'on_repairment': 'onRepairParts',
+
+
+            Returns:
+                JSON-formatted output with data as described here:
```

Comment:
As we discussed somewhere - this is not JSON-formatted output, but python dictionary, right?